test/e2e/app-dir/mdx/next.config.js

```diff
@@ -8,8 +10,23 @@ const withMDX = require('@next/mdx')({
 const nextConfig = {
   pageExtensions: ['js', 'jsx', 'ts', 'tsx', 'mdx'],
   experimental: {
-    mdxRs: process.env.WITH_MDX_RS === 'true',
+    mdxRs: shouldUseMdxRs,
   },
 }
 
+if (shouldUseMdxRs) {
```

Comment:
Shouldn't turbopack automatically apply this if `experimental.mdxRs` is enabled?

Reply:
mdxRS shouldn't need a loader at all since it's using the rust mdx version